searchlib/src/vespa/searchlib/fef/indexproperties.cpp

```diff
@@ -49,10 +50,12 @@ uint32_t
 lookupUint32(const Properties &props, const vespalib::string &name, uint32_t defaultValue)
 {
     Property p = props.lookup(name);
+    uint32_t value(defaultValue);
     if (p.found()) {
-        return atoi(p.get().c_str());
+        const auto & s = p.get();
+        std::from_chars(s.c_str(), s.c_str() + s.size(), value);
```

Comment:
Do we want to check the `error_code` returned as part of the result tuple, or is the fallback to `defaultValue` considered good enough?

Reply:
We have no way of handling any error, so fallback to default value is the best we can do I think.
One step better than atoi.